```
pkg/validation/user.go:104:6: SA6005: should use strings.EqualFold instead (staticcheck)
		if strings.ToLower(user.Spec.Email) == strings.ToLower(email) {
		   ^
pkg/webhook/user/validation/validation.go:93:5: SA6005: should use strings.EqualFold instead (staticcheck)
	if strings.ToLower(oldUser.Spec.Email) != strings.ToLower(newUser.Spec.Email) {
	   ^
2 issues:
* staticcheck: 2
```

--- a/pkg/validation/user.go
+++ b/pkg/validation/user.go
@@ -101,7 +101,7 @@ func ValidateUserEmailUniqueness(ctx context.Context, client ctrlruntimeclient.C
 			continue
 		}
 
-		if strings.ToLower(user.Spec.Email) == strings.ToLower(email) {
+		if strings.EqualFold(user.Spec.Email, email) {
 			return field.Duplicate(field.NewPath("spec", "email"), email)
 		}
 	}

--- a/pkg/webhook/user/validation/validation.go
+++ b/pkg/webhook/user/validation/validation.go
@@ -90,7 +90,7 @@ func (v *validator) ValidateUpdate(ctx context.Context, oldObj, newObj runtime.O
 		errs = append(errs, err)
 	}
 
-	if strings.ToLower(oldUser.Spec.Email) != strings.ToLower(newUser.Spec.Email) {
+	if !strings.EqualFold(oldUser.Spec.Email, newUser.Spec.Email) {
 		if err := validation.ValidateUserEmailUniqueness(ctx, v.client, newUser.Spec.Email, newUser.Name); err != nil {
 			errs = append(errs, err)
 		}

